pkg/operator/reporting/validate.go

```diff
@@ -132,8 +124,7 @@ type reportGenerationQueryDependenciesValidationError struct {
 	uninitializedQueryNames,
 	disabledViewQueryNames,
 	uninitializedDataSourceNames,
-	uninitializedReportNames,
-	uninitializedScheduledReportNames []string
+	uninitializedReportNames []string
```

Comment:
Why unititalizedReportNames need string slice now?

Reply:
It was always a string slice. Nothing changed. We just removed the `uninitializedScheduledReportNames` field.